server/schema/contests/updateContest.js

```diff
@@ -46,12 +46,12 @@ module.exports = {
       type: 'boolean',
     },
     start_time: {
-      type: 'object',
-      format: 'date-time',
+      type: 'string',
+      format: 'dateTimeFormat',
     },
     end_time: {
-      type: 'object',
-      format: 'date-time',
+      type: 'string',
+      format: 'dateTimeFormat',
```

Comment:
custom validator in `updateContest` schema also changed, same as `createContest` schema

Reply:
cool